src/SmartComponents/AdvisorySystems/AdvisorySystem.test.js

```diff
@@ -89,7 +89,12 @@ afterEach(() => {
 describe('AdvisorySystems.js', () => {
 
     it('Should match the snapshots and dispatch FETCH_AFFECTED_SYSTEMS only once', () => {
```

Comment:
This test here and the snapshot is something we should replace with something other than `asFragment`.

The old test was already kind of useless. However, the title here does suggest a better way to test here.

Instead of trying to render, we could verify that it dispatches the right action and fetch (and only once) and to make it even better we should verify it gets dispatched for the correct advisory name. 

We could also try to not mock too much and let it all run through and verify via an rtl query that a element with the correct advisory name is present on screen.

Reply:
I agree, I tried to replace the asFragment with something that testing the actual inventory table that is rendered. 
I tried the following
```
render (our component)
and then use
screen.debug() to log the DOM.
```
This is what I get in the terminal, why the table is not rendering? Do you have any idea?
![image](https://github.com/RedHatInsights/patchman-ui/assets/62722417/111340a8-a062-4ccf-a2f4-8762fb80935a)

Reply:
Are you providing the necessary data for it to render the correct state?